setup.py

```diff
@@ -33,20 +33,20 @@ def get_version():
 
 # Environment-specific dependencies.
 extras = {
-    "atari": ["ale-py~=0.8.0"],
+    "atari": ["shimmy[atari]>=0.1.0,<1.0"],
     "accept-rom-license": ["autorom[accept-rom-license]~=0.4.2"],
     "box2d": ["box2d-py==2.3.5", "pygame==2.1.0", "swig==4.*"],
     "classic_control": ["pygame==2.1.0"],
     "mujoco_py": ["mujoco_py<2.2,>=2.1"],
-    "mujoco": ["mujoco==2.2", "imageio>=2.14.1"],
+    "mujoco": ["mujoco==2.3", "imageio>=2.14.1"],
     "toy_text": ["pygame==2.1.0"],
     "jax": ["jax==0.3.20", "jaxlib==0.3.20"],
     "other": ["lz4>=3.1.0", "opencv-python>=3.0", "matplotlib>=3.0", "moviepy>=1.0.0"],
 }
 
 extras["testing"] = list(set(itertools.chain.from_iterable(extras.values()))) + [
     "pytest==7.1.3",
-    "gym[classic_control, mujoco_py, mujoco, toy_text, other, atari, accept-rom-license]==0.26.2",
```

Comment:
Previously, this was necessary more for installing atari than any of the other environments. 
I believe we still have tests that the compatibility environment can be accessed otherwise, we could remove this test

Reply:
> we could remove this test

Was this meant for another comment? I was just wondering about the difference between installing `gym[...]` vs `gym`, does this do the same thing, or does it do something different but we don't actually care?

Reply:
I think it should do the same thing in this PR, in previous PRs different as atari wasn't installed. 
But now that shimmy deals with compatibility now, we should be able to remove the tests